<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringParameterView</class>
 <widget class="QFrame" name="StringParameterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>265</width>
     <height>290</height>
    </rect>
   </property>
   <property name="title">
    <string>String Parameter</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="numberText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="descriptionText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>45</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="identifierText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifier</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isWriteableBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Writeable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isCommandCheckBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Command</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>valueText</sender>
   <signal>editingFinished()</signal>
   <receiver>StringParameterView</receiver>
   <slot>updateEntityValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isCommandCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>StringParameterView</receiver>
   <slot>updateEntityCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descriptionText</sender>
   <signal>editingFinished()</signal>
   <receiver>StringParameterView</receiver>
   <slot>updateEntityDescription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isWriteableBox</sender>
   <signal>clicked()</signal>
   <receiver>StringParameterView</receiver>
   <slot>updateEntityWriteable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateEntityDescription()</slot>
  <slot>updateEntityValue()</slot>
  <slot>updateEntityMax()</slot>
  <slot>updateEntityWriteable()</slot>
  <slot>updateEntityCommand()</slot>
 </slots>
</ui>
